First Examples for Samples API Usage in Java

MalikAtalla-AI opened the pull request:
@jessemcdowell-AI : Please have a look

These are not the examples we have discussed for the training. I think those may be too specific for this repo. This is a simpler set of examples, I could show in the training as a starting point for the participants. Basing off of this, they could try to implement the analytical groups importer we discussed.

jessemcdowell-AI commented:
How about AQUARIUS Samples/API Examples/

jessemcdowell-AI commented on .gitignore:
```diff
@@ -0,0 +1,3 @@
+.idea
```
Let's move this into the AQUARIUS Samples directory

MalikAtalla-AI replied:
Done

jessemcdowell-AI commented on AQUARIUS Samples API Examples/src/main/java/ai/training/ObservedPropertyExample.java:
```diff
@@ -0,0 +1,42 @@
+            System.out.println("Usage:\n" +
+                    "java AnalyticalGroupsImporter <AQ Samples URL> <TOKEN>\n\n" +
+                    "For Example:\n" +
+                    "java AnalyticalGroupsImporter https://mycompany.aqsamples.net/api/v1/ 054203b73b913a6fe5bc8d9da425dff9");
```
should be aqsamples.com

MalikAtalla-AI replied:
Right. Thanks

jessemcdowell-AI commented on AQUARIUS Samples API Examples/src/main/java/ai/training/ObservedPropertyExample.java:
```diff
@@ -0,0 +1,42 @@
+    public static void main(String[] args) {
+        if (args.length != 2) {
+            System.out.println("Usage:\n" +
+                    "java AnalyticalGroupsImporter <AQ Samples URL> <TOKEN>\n\n" +
```
This class has more to do with observed properties than analytical groups

MalikAtalla-AI replied:
Changed

jessemcdowell-AI commented on AQUARIUS Samples API Examples/src/main/java/ai/training/AqSamplesClient.java:
```diff
@@ -0,0 +1,60 @@
+        return response.readEntity(new GenericType<List<UnitGroup>>() { });
+    }
+
+    public List<ObservedProperty> getObservedProperties() {
```
This would be more helpful if you returned a `Map<String,ObservedProperty>` where the key of the map is the custom id of the observed property. The only reason someone would use this method is when they need the object for composing new objects, and they will likely want to look them up by custom id. Same for `getUnitGroups()`.

MalikAtalla-AI replied:
I'm not a 100% convinced that it's a good idea, since the purpose of this piece of code is not really to be useful, but rather to demonstrate the usage of the API in a simple way. But I added it anyway mostly to use the fancy Java 8 streams.

MalikAtalla-AI commented:
@jessemcdowell-AI : I addressed all your comments. Could you have a look? I just noticed I kind of need this merged today, but you're probably not available. Not sure if somebody else can merge this. It looks like you put some restrictions around merging on this repo.
